pyanaconda/modules/services/services.py

```diff
@@ -80,6 +83,15 @@ def process_kickstart(self, data):
         setup_on_boot = self._map_firstboot(data.firstboot.firstboot)
         self.set_setup_on_boot(setup_on_boot)
 
+        # Having "firstboot --disable" in the input kickstart
+        # sets post_install_tools_disable property,
+        # which is then reflected in the user interaction config file
+        # parsed by Gnome Initial Setup.
+        # This makes it possible to disable both with a single
+        # kickstart command.
```

Comment:
Please, make a nice text block from this.

Reply:
The block has been improved.